cmds/coredhcp/main.go

```diff
@@ -31,14 +29,15 @@ import (
 	pl_serverid "github.com/coredhcp/coredhcp/plugins/serverid"
 
 	"github.com/sirupsen/logrus"
+	flag "github.com/spf13/pflag"
 )
 
 var (
-	flagLogFile     = flag.String("logfile", "", "Name of the log file to append to. Default: stdout/stderr only")
-	flagLogNoStdout = flag.Bool("nostdout", false, "Disable logging to stdout/stderr")
-	flagLogLevel    = flag.String("loglevel", "info", fmt.Sprintf("Log level. One of %v", getLogLevels()))
-	flagConfig      = flag.String("conf", "", "Use this configuration file instead of the default location")
-	flagPlugins     = flag.Bool("plugins", false, "list plugins")
+	flagLogFile     = flag.StringP("logfile", "l", "", "Name of the log file to append to. Default: stdout/stderr only")
+	flagLogNoStdout = flag.BoolP("nostdout", "N", false, "Disable logging to stdout/stderr")
+	flagLogLevel    = flag.StringP("loglevel", "L", "info", fmt.Sprintf("Log level. One of %v", getLogLevels()))
```

Comment:
maybe those options aren't worth a short option ? I'm not sure they're used that often

Reply:
I am personally in favour of always having both unless an option is really exotic and we should discourage its use. Which ones would you remove?

Reply:
probably `-N` and `-l` but having a policy that everything has a shortflag is fine too

Reply:
merging as discussed